src/vcpkg/vcpkgpaths.cpp

```diff
@@ -330,22 +314,18 @@ namespace vcpkg
                 const auto status = get_real_filesystem().status(ret, VCPKG_LINE_INFO);
                 if (!vcpkg::exists(status))
                 {
-                    Checks::exit_with_message(VCPKG_LINE_INFO,
-                                              "Path to X_VCPKG_REGISTRIES_CACHE does not exist: " + ret.native());
+                    Checks::msg_exit_with_message(
+                        VCPKG_LINE_INFO, msgMissingPathToRegistryCache, msg::path = ret.native());
                 }
 
```

Comment:
I think we should just delete this block entirely; `is_directory` already implies an exists check.